store/sqlstore/channel_store.go

```diff
@@ -830,12 +847,12 @@ func (us SqlChannelStore) IsUserInChannelUseCache(userId string, channelId strin
 			return false
 		}
 	} else {
-		if us.metrics != nil {
-			us.metrics.IncrementMemCacheMissCounter("All Channel Members for User")
+		if s.metrics != nil {
+			s.metrics.IncrementMemCacheMissCounter("All Channel Members for User")
```

Comment:
nit: Worth considering a `const` declaration for these vs. duplicating the strings and potentially typoing?

Reply:
If I wasn't so busy I'd do it right now, but you're right so I at least made a ticket to do it in the future https://mattermost.atlassian.net/browse/MM-9670